Libraries/Components/TextInput/TextInput.flow.js

```diff
@@ -828,6 +828,13 @@ export type Props = $ReadOnly<{|
    */
   onContentSizeChange?: ?(e: ContentSizeChangeEvent) => mixed,
 
+  /**
+   * Callback that is called when the text is input into the text input.
+   * The original text, the selection in the original text, and the inserted
+   * text are passed to the callback handler.
+   */
+  onTextInput?: ?(e: TextInputEvent) => mixed,
+
```

Comment:
Since it is an addition (I know it was removed upstream), can we add `// [macOS]` tags

Reply:
Do we also need to update the typescript definition?

Reply:
Removed per offline discussion.